supabase/migrations/20230619041056_users.sql

```diff
@@ -3,6 +3,7 @@ create table "public"."users" (
     "created_at" timestamp with time zone default now(),
     "name" text not null,
     "age" smallint,
+    "phone_number" varchar(8),
```

Comment:
I think we should just use `text` here to not limit us too much, e.g. might want to add country codes in the future etc.

Reply:
hmm ok